### PR TITLE
Main menu screen & bug fixes

### PR DESCRIPTION
**Pull request by lukybil**
- resolved comments from the previous PR (I did not change the SearchContext because I did not want to end up with a function `setSetFlyToCenter`, I can update it if necessary)
- #220 newlines in markdown fix
- #272 navigate to current location button: If permission is denied, the button is hidden, if location service is off, after pressing the button, location should be requested to be turned on
- #241 main menu as screen
- #231 text overlap fix

Points for a discussion:
- Should the main menu have a fixed distance?
- What should be the size of the spacer on the left? Right now the menu has a min width and is set to flex grow
- Should the spacer be a backdrop? Should it have a lets say `rgba(0,0,0,0.5)` background?

### CODE REVIEW

**Review by PavelFilo — APPROVED** [review]

File: components/shared/Markdown.tsx [hunk 1]
```diff
@@ -41,7 +61,11 @@ type Props = {
 const Markdown = ({ children, fontSize = 'default', textCenter }: Props) => {
   const rules = useMemo(() => getRules(fontSize, textCenter), [fontSize, textCenter])
 
-  return <MarkdownNative rules={rules}>{children}</MarkdownNative>
+  return (
+    <MarkdownNative rules={rules} markdownit={markdownInstance} debugPrintTree>
```
Comment (radoslavzeman) [inline components/shared/Markdown.tsx:65]: Do we want `debugPrintTree` for prod?

**Review by radoslavzeman — APPROVED** [review]
Please double check all `nearBy`/`NearBy` occurrences and use `b` instead of `B` :D

**Comment by radoslavzeman** [comment]
Questions can be addressed in Figma and solved as separate PR.

**Comment by radoslavzeman** [comment]
Safe view should be used in multiple places, it's a good point, but I would keep it as separate issue.

**Pull request merged**